cme/lms/templates/theme-settings.html

```diff
@@ -1,16 +0,0 @@
-## theme-settings.html
-
-<%!
-  theme_settings_vars = {
-    'effective_date': 'October 26, 2015',
-    'accessibility_email': '<a href="mailto:accessibility@stanfordonline.zendesk.com">accessibility@stanfordonline.zendesk.com</a>',
-    'concern_email': '<a href="mailto:concern@stanfordonline.zendesk.com">concern@stanfordonline.zendesk.com</a>',
-    'privacy_email': '<a href="mailto:privacy@stanfordonline.zendesk.com">privacy@stanfordonline.zendesk.com</a>',
```

Comment:
Where is this file used again?
Are we sure we want to use all the same values?

Reply:
These values get used in the contact page and ToS, Honor, Privacy Policy pages. Since all those pages are symlinked, I symlinked this file as well...